packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -551,6 +554,7 @@ class BottomSheet extends Component {
 					<View
 						style={ styles.header }
 						onLayout={ this.onHeaderLayout }
+						testID="bottom-sheet-header"
```

Comment:
I'm wondering if we should try to keep test IDs unique to make queries easier to be made in tests. Maybe we could use a suffix like `-header` in case the `testID` prop is passed:
 ```suggestion
						testID={ `${ rest.testID || 'bottom-sheet' }-header` }
```

Wdyt?

Reply:
There doesn't appear to be an explicit need for this currently, but I think this change is sensible to include. 👍🏻

Reply:
@dcalhoun I'm afraid I provided a bad indentation on the suggested change 😭, I'll push a fix to address it asap.

Reply:
I've just solved it in https://github.com/WordPress/gutenberg/pull/37782/commits/3fdd6b3f64ef8462866636e76a9102389b6877e7, sorry for the mistake.